### DISCUSSION TITLE
1.9" st7789 170x320 display

### DISCUSSION THREAD

**Discussion by pipikupi** [Bodmer/TFT_eSPI#3471] [Compatible displays and setup files]
I've got display from ali...s
https://www.tztstore.com/goods/show-6316.html
Tried to set that with ESP32 using standard pins
#define TFT_MISO 19  // not connected
#define TFT_MOSI 23  
#define TFT_SCLK 18

#define TFT_CS   05
#define TFT_DC   16
#define TFT_RST  17 // EN 
but there is still nothing working.
Will be appreciated for any advice.
Tried few setups for that chip and none of them is working

**Comment by stef-ladefense** [comment]
-DST7789_DRIVER=1
-DTFT_WIDTH=170
-DTFT_HEIGHT=320
-DTFT_MISO=-1
-DTFT_MOSI=13
-DTFT_SCLK=12
-DTFT_CS=10
-DTFT_DC=11
-DTFT_RST=1
-DTOUCH_CS=-1
-DTFT_BL=14
-DTFT_BACKLIGHT_ON=1
-DLOAD_GLCD=0
-DLOAD_FONT2=1
-DLOAD_FONT4=1
-DLOAD_FONT6=0
-DLOAD_FONT7=1
-DLOAD_FONT8=0
-DLOAD_GFXFF=0
-DSMOOTH_FONT=0
-DSPI_FREQUENCY=30000000
-DSUPPORT_TRANSACTIONS=0
-DUSE_HSPI_PORT=1
-DDISABLE_ALL_LIBRARY_WARNINGS=0
